components/builder-scheduler/habitat/default.toml

```diff
@@ -1,3 +1,8 @@
+auth_token = ""
+depot_url = ""
+auto_promote_channel = "unstable"
+log_path = "/hab/svc/builder-scheduler/logs"
```

Comment:
Since this is something we're dynamically generating and we don't actually want overridden, I'd put this bit into the template itself at `builder-scheduler/habitat/config/config.toml`. There's some samples of this pattern [here](https://github.com/habitat-sh/habitat/blob/master/components/builder-api/habitat/config/config.toml#L1)

It's holes like this which keep making me think that toml isn't the right language to represent our configuration interface